src/v3/src/components/List/List.tsx

```diff
@@ -52,12 +50,6 @@ const renderElement = (item: UISchemaElement) => {
           <InformationalText uischema={item as DescriptionElement} />
         </Container>
       );
-    case 'TextWithHtml':
-      return (
-        <Container>
-          <TextWithHtml uischema={item as TextWithHtmlElement} />
-        </Container>
-      );
```

Comment:
For now this is no longer needed, the ODA screen that used this is now using the standard Description (informational text) component (which contains the parser logic) so the anchor link will get transformed correctly (updated screenshot attached to PR).